### PR TITLE
LE Audio: BSIM broadcast AC testing

### LINKED ISSUE #64245: LE Audio: Expand CAP BSIM tests with data validation
**Is your enhancement proposal related to a problem? Please describe.**
The tests are currently verifying that the CIS/BIS and can be established, but we never test if data can actually be sent with the providing configurations. 

**Describe the solution you'd like**
Add data validation (similar to https://github.com/zephyrproject-rtos/zephyr/pull/62496) to the CAP AC tests to test that it actually works. 

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Thalley**
Adds testing of the audio configuration for the low latency presets. 

fixes https://github.com/zephyrproject-rtos/zephyr/issues/64245

### CODE REVIEW

**Review by cvinayak — APPROVED** [review]
QUICK ACK based on using the tests for analyzing controller bugs

File: tests/bsim/bluetooth/audio/src/gmap_ugg_test.c [hunk 1]
```diff
@@ -23,35 +23,44 @@
...
+#define GMAP_UNICAST_AC_MAX_SNK      (2U * GMAP_UNICAST_AC_MAX_CONN)
+#define GMAP_UNICAST_AC_MAX_SRC      (2U * GMAP_UNICAST_AC_MAX_CONN)
+#define GMAP_UNICAST_AC_MAX_PAIR     MAX(GMAP_UNICAST_AC_MAX_SNK, GMAP_UNICAST_AC_MAX_SRC)
+#define GMAP_UNICAST_AC_MAX_STREAM   (GMAP_UNICAST_AC_MAX_SNK + GMAP_UNICAST_AC_MAX_SRC)
 
-#define MAX_CIS_COUNT     2U
+#define MAX_ISO_CHAN_COUNT 2U
 #define ISO_ENQUEUE_COUNT 2U
-#define TOTAL_BUF_NEEDED  (ISO_ENQUEUE_COUNT * MAX_CIS_COUNT)
+#define TOTAL_BUF_NEEDED       (ISO_ENQUEUE_COUNT * MAX_ISO_CHAN_COUNT)
```
Comment (larsgk) [inline tests/bsim/bluetooth/audio/src/gmap_ugg_test.c:35]: alignment?

**Review by larsgk — APPROVED** [review]
LGTM - my eye didn't catch anything critical, but didn't have time to run the tests

File: tests/bsim/bluetooth/audio/src/gmap_ugg_test.c (hunk 1 above)
Comment (Thalley) [reply tests/bsim/bluetooth/audio/src/gmap_ugg_test.c:35]: #blameclangformat :D

**Pull request merged**